### PR TITLE
Update jwt-decoder extension

### PR DESCRIPTION
**Pull request by zhhanging**
## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
Thanks for your contribution @zhhanging 🔥 Do you mind adding a [CHANGELOG](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) entry too?

@gdsmith can you approve this?

**Comment by zhhanging** [comment]
I have Added a CHANGELOG.md file as suggested.

**Comment by pernielsentikaer** [comment]
@gdsmith are you around?

File: extensions/jwt-decoder/src/jwt-view.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const JwtView = () => {
     const headItems = ListFromObject(header, claims);
     const dataItems = ListFromObject(data, claims);
 
-    const tokenImg = `<img alt="view token" width="720" src="data:image/svg+xml,${encodeURI(
+    const tokenImg = `<img alt="view token" width="720" src="data:image/svg+xml,${encodeURIComponent(
```
Comment (gdsmith) [inline extensions/jwt-decoder/src/jwt-view.tsx:41]: There's another use of encodeURI in `jwt-list.tsx` can you replace that one too?

**Review by gdsmith — CHANGES_REQUESTED** [review]
@zhhanging thanks so much for fixing this, I mostly use the list view so hadn't noticed this had broken. Any chance you can update the other `encodeURI` use as per my comment?

File: extensions/jwt-decoder/src/jwt-view.tsx (hunk 1 above)
Comment (zhhanging) [reply extensions/jwt-decoder/src/jwt-view.tsx:41]: I have replaced the encodeURI in `jwt-list.tsx` as suggested. Thank you.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**